### DISCUSSION TITLE
how we can added custom button side by filter

### DISCUSSION THREAD

**Discussion by deepak458** [rappasoft/laravel-livewire-tables#983] [Q&A] [answered]
if we want to add custom filter for create or any other how we can addd

**Comment by lrljoe** [comment, marked as answer]
Look at configurable areas docs, otherwise publish the views and you can add whatever you like in there.

Let me know if this needs clarifying

**Comment by deepak458** [comment]
can you please help me with link of configurable areas page for this, i have tried but didn't found any relevent answer.

also if i publish view that will update changes for all views ?

**Comment by deepak458** [comment]
i took my last comment back, yes this is working fine and i'm very thanksful for your answer
